app/controllers/applications_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ApplicationsController < ApplicationController
+  def show
+    @applications = Application.find(params[:id])
+  end
+
+  def new
+    # @application = Application.new
+  end
+
+  def create
+    @application = Application.new({ name: params[:name], address: params[:address],
+                                     city: params[:city], state: params[:state], zip: params[:zip],
+                                     description: params[:description], status: 'In Progress' })
+    if @application.save
```

Comment:
I think this was a good way to solve this issue. Great work changing create to new so that we can utilize .save in the conditional.